lib/translator/index.js

```diff
@@ -63,6 +63,11 @@ function Translator(acceptLanguage, allTranslations, logger = console) {
  * @returns {Translator}
  */
 Translator.create = function (acceptLanguage, allTranslations, logger = console) {
+    //  Safari only sends the most preferred language's code, this is to have fallbacks in case we don't have that language
+    //  As an example we may not have fr-FR so add fr to the header
+    if (acceptLanguage.split(' ').length === 1 && acceptLanguage.split('-').length === 2) {
```

Comment:
This kind of logic would be better placed in `lib/translator/locale-parser`, perhaps in the `getLocales` function. You could examine the output of `AcceptLanguageParser.parse` and then update as necessary, rather than modify the header value here.

Reply:
fixed.